### ISSUE TITLE
Non related to ViaVersion but can be fixed by them (I believe)

### ISSUE THREAD

**Issue by camdenorrb** [MylesIsCool/ViaVersion#353]
### What version of ViaVersion are you using? Type /ver viaversion
0.7.5
### What version of Spigot are you using? Type /ver
1.8.8
### What plugins are you using? Type /plugins
All custom plus via version
### Are you using any additional software like BungeeCord? If so, what software and version? (Please list your plugins as well)
latest
### How does this error happen? login? Using an item? 
No errors
### Is there an error in the console? Use pastebin.com. Is there a kick message?
Nope.

Basically the problem is when you have a void world and break blocks above void it appears to try to update light which can damper performance. I know this isn't caused by ViaVersions but you guys might be able to fix it through it. Thanks for reading my suggestion :3

**Comment by MylesIsCool** [comment]
We can't fix this. We live at a packet level, it's a known issue with Minecraft.

If you want to edit spigot or what to do it then sure. We have no plans to.
